assets/demo.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  required_providers {
+    iterative = {
+      versions = ["0.1"]
+      source = "github.com/iterative/iterative"
```

Comment:
Do we have this repo\url?

Reply:
no its inherited from the hashicorp tutorial that sets that bad. I will fix it

Reply:
its a terraform thingy. After fixing it

![image](https://user-images.githubusercontent.com/414967/92538114-26bc7300-f23e-11ea-9bf2-66a1b6af33e4.png)